coordinator/src/main/scala/filodb.coordinator/queryplanner/DefaultPlanner.scala

```diff
@@ -338,17 +342,24 @@ trait  DefaultPlanner {
                                         sqww: SubqueryWithWindowing
                                       ) : PlanResult = {
     // absent over time is essentially sum(last(series)) sent through AbsentFunctionMapper
-    innerExecPlan.plans.foreach(
-      _.addRangeVectorTransformer(PeriodicSamplesMapper(
-        sqww.startMs, sqww.stepMs, sqww.endMs,
-        window,
-        Some(InternalRangeFunction.lpToInternalFunc(RangeFunctionId.Last)),
-        qContext,
-        stepMultipleNotationUsed = false,
-        Seq(),
-        offsetMs,
-        rawSource = false
-      ))
+     val realStartMs = sqww.atMs.getOrElse(sqww.startMs)
+     val realEndMs = sqww.atMs.getOrElse(sqww.endMs)
+     val realStep = sqww.atMs.map(_ => 0L).getOrElse(sqww.stepMs)
```

Comment:
Why is `atMs` used to get the step?

Reply:
the value of atMs is not used for step. when atMs is provided step should be 0 to turn the query an instant query.